control.lua

```diff
@@ -108,6 +110,7 @@ local function onInit()
     global.world = {}
 
     world = global.world
+    minDiffuse = game.map_settings.pollution.min_to_diffuse
```

Comment:
why create this as a separate variable and not use the world global that was already created?